pages/article/[id].js

```diff
@@ -139,104 +228,141 @@ function ArticlePage() {
+                articleId={article.id}
+                onNewReplyButtonClick={() => {
+                  setShowForm(true);
+                }}
+              />
+            </footer>
+          </Box>
+
+          {showForm && (
+            <div className={classes.newReplyContainer}>
+              <NewReplySection
+                article={article}
+                existingReplyIds={(article?.articleReplies || []).map(
+                  ({ replyId }) => replyId
+                )}
+                relatedArticles={article?.relatedArticles}
+                onSubmissionComplete={handleNewReplySubmit}
+                onClose={() => setShowForm(false)}
+              />
+            </div>
           )}
-          relatedArticles={article?.relatedArticles}
-          onSubmissionComplete={handleNewReplySubmit}
-        />
-      </section>
-
-      {article?.similarArticles?.edges?.length > 0 && (
-        <section className="section">
-          <h2>{t`You may be interested in the following similar messages`}</h2>
-          <ul className="similar-articles">
-            {article.similarArticles.edges.map(({ node }) => (
-              <ArticleItem key={node.id} article={node} />
-            ))}
-          </ul>
-        </section>
-      )}
-
-      <style jsx>{`
-        .section {
-          margin-bottom: 64px;
-        }
-        .header {
-          display: flex;
-          align-items: center;
-          flex-flow: row wrap;
-        }
-        .header > .trendline {
-          margin: 0 16px 0 auto;
-        }
-        .message {
-          border: 1px solid #ccc;
-          background: #eee;
-          border-radius: 3px;
-          padding: 24px;
-          word-break: break-all;
-        }
-        .items {
-          list-style-type: none;
-          padding-left: 0;
-        }
-        .similar-articles {
-          padding: 0;
-          list-style: none;
-        }
-      `}</style>
+
+          <Box
+            className={classes.card}
+            position="relative"
+            px={{ xs: '12px', md: '19px' }}
+            py={{ xs: '13px', md: '21px' }}
+            mt={3}
+            id="current-replies"
+            ref={replySectionRef}
+          >
+            <h2>{t`${article.articleReplies.length} replies to the message`}</h2>
+            <Divider classes={{ root: classes.divider }} />
+            <CurrentReplies articleReplies={article.articleReplies} />
+          </Box>
+        </div>
+
+        <div className={cx(classes.card, classes.aside)}>
+          <h4>{t`Similar messages`}</h4>
+          {similarArticles.length ? (
+            <Box
+              display="flex"
+              flexDirection={{ xs: 'row', md: 'column' }}
+              overflow="auto"
+            >
+              {similarArticles.map(({ node }) => (
+                <Link key={node.id} href={`/article/${node.id}`}>
+                  <div className={classes.similarMessageContainer}>
```

Comment:
Next.js [suggests](https://nextjs.org/docs/api-reference/next/link) using `<a>` tag for `<Link>`'s children. I think we can directly change `.similarMessageContainer` to use `a` tag instead of `div`.

Currently it do navigates user if they click the link, but if `<Link>` is combined with `<a>`, it allows user to cmd-click to open the link in new tab. We won't need `cursor: pointer` on the element as well, since `<a>` already turns cursor into pointer. (But we may need `text-decoration: none` to cancel out `<a>`'s default style, though.)